packages/core/src/configuration/Configuration.spec.ts

```diff
@@ -0,0 +1,43 @@
+import 'reflect-metadata';
+import { Configuration } from './Configuration';
+import { EnvCredentialProvider } from '../credentials-provider';
+import { instance, mock, reset, verify, when } from 'ts-mockito';
+
+describe('configuration', () => {
+  it('should be a single instance', () => {
+    const configuration = new Configuration({});
+    const configuration2 = configuration.container.resolve(Configuration);
+    expect(configuration).toBe(configuration2);
+  });
+
+  describe('loadCredentials', () => {
+    const mockedProvider = mock<EnvCredentialProvider>();
+
+    afterEach(() => {
+      reset(mockedProvider);
+    });
```

Comment:
Remove redundant brackets

```suggestion
    afterEach(() => reset(mockedProvider));
```